.eslintrc

```diff
@@ -38,7 +38,8 @@
                 "args": "none"
             }
         ],
-        "comma-dangle": 0,
+        "no-console": 1,                      // 1=warn
+        "comma-dangle": 0,                    // 0=off
```

Comment:
Take this example within [LINT.md](https://github.com/cyverse/troposphere/blob/master/LINT.md#no-unused-vars):
```
"no-unused-vars": [2, {"args": "none"}]
```